releasenotes/notes/31695-releasenote.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+
+issue:
+  - https://github.com/istio/istio/issues/31695
+
+docs:
+ - '[usage] https://istio.io/latest/docs/reference/commands/istioctl/#istioctl-proxy-config'
+
+releaseNotes:
+- |
+  **Added** `istioctl proxy-config -o yaml` to display in yaml along with the current json and short format
+
+upgradeNotes:
```

Comment:
These fields are not required. Please refer https://github.com/istio/istio/blob/master/releasenotes/notes/18487.yaml.